src/style.css

```diff
@@ -0,0 +1,20 @@
+#cell1 {
+  background-color: red;
+  padding: 5%;
+  margin: -8px;
+  height: 5px;
+}
+
+#cell2 {
+  background-color: green;
+  padding: 5%;
+  margin: -8px;
+  height: 5px;
+}
+
+#cell3 {
+  background-color: blue;
+  padding: 5%;
+  margin: -8px;
+  height: 5px;
+}
```

Comment:
If we have 5 blocks and these blocks have common characteristics, such as height, width, color, fonts, etc., then we should move these characteristics in a separate class and use it for style these blocks. DRY - Don't Repeat Yourself.